src/main/java/frc/trigon/robot/subsystems/MotorSubsystem.java

```diff
@@ -59,6 +76,10 @@ public static void setAllSubsystemsBrakeAsync(boolean brake) {
         CompletableFuture.runAsync(() -> forEach((subsystem) -> subsystem.setBrake(brake)));
     }
 
+    public static boolean isExtensiveLoggingEnabled() {
```

Comment:
Static functions should be above non-static ones.

Reply:
It is

Reply:
The periodic should be below it.